guten_reader_FE/src/Components/Settings/Settings.js

```diff
@@ -0,0 +1,91 @@
+import React, { Component } from 'react';
+import { View, Text, Picker, StyleSheet } from 'react-native';
+
+class Settings extends Component {
+  constructor() {
+    super();
+    this.state = {
+      musicGenre: 'classical',
+      font: 'roboto',
+      fontSize: '20',
+      darkMode: 'false'
+    }
+  }
+
+  render() {
+    return (
+      <View style={styles.container}>
+        <Text style={styles.text}>Settings Component</Text>
+        <Text style={styles.text}>Genre: {this.state.musicGenre}</Text>
+        <Text style={styles.text}>font: {this.state.font}</Text>
+        <Text style={styles.text}>fontSize: {this.state.fontSize}</Text>
+        <Text style={styles.text}>darkMode: {this.state.darkMode}</Text>
+
+        <View style={styles.pickerContainer}>
+          <Picker
+            style={styles.picker}
+            selectedValue={this.state.musicGenre}
+            onValueChange={(itemValue) =>
+              this.setState({ musicGenre: itemValue})
+          }>
+          <Picker.Item label="Classical" value="classical" />
```

Comment:
These pickers are super cool, easy to use, and provide great consistency in style/use... great choice!